src/test/java/com/atlassian/jira/cloud/jenkins/common/client/JenkinsAppRequestTestData.java

```diff
@@ -0,0 +1,67 @@
+package com.atlassian.jira.cloud.jenkins.common.client;
+
+import com.atlassian.jira.cloud.jenkins.buildinfo.client.model.Builds;
+import com.atlassian.jira.cloud.jenkins.buildinfo.client.model.JiraBuildInfo;
+import com.atlassian.jira.cloud.jenkins.buildinfo.client.model.TestInfo;
+import com.atlassian.jira.cloud.jenkins.deploymentinfo.client.model.Deployments;
+import com.atlassian.jira.cloud.jenkins.deploymentinfo.client.model.Environment;
+import com.atlassian.jira.cloud.jenkins.deploymentinfo.client.model.JiraDeploymentInfo;
+import com.atlassian.jira.cloud.jenkins.deploymentinfo.client.model.Pipeline;
+
+import java.time.Instant;
+import java.util.Arrays;
+import java.util.Collections;
+import java.util.HashSet;
+
+public class JenkinsAppRequestTestData {
```

Comment:
Extracted the generation of test objects into this class to be used from multiple places.